src/v3/src/util/formUtils.ts

```diff
@@ -55,6 +56,25 @@ export const handleFormFieldBlur = (
   setUntouchedFn?.(false);
 };
 
+export const getCustomButtonElements = (widgetProps: WidgetProps): ButtonElement[] => {
+  const { customButtons = [] } = widgetProps;
+  return customButtons.map((customButton) => {
+    const customButtonElement: ButtonElement = {
+      type: 'Button',
+      label: customButton.title ?? loc(customButton.i18nKey, 'login'),
+      options: {
+        type: ButtonType.BUTTON,
+        step: '',
+        dataSe: customButton.dataAttr,
+        classes: classNames(customButton.className, 'default-custom-button'),
```

Comment:
@glenfannin-okta do we have any compatibility issues with `classNames` or is it fine to use it like this?